packages/schematics/angular/migrations/update-8/update-lazy-module-paths_spec.ts

```diff
@@ -0,0 +1,78 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+// tslint:disable:no-big-function
+import { EmptyTree } from '@angular-devkit/schematics';
+import { SchematicTestRunner, UnitTestTree } from '@angular-devkit/schematics/testing';
+
+describe('Migration to version 8', () => {
+  const schematicRunner = new SchematicTestRunner(
+    'migrations',
+    require.resolve('../migration-collection.json'),
+  );
+
+  let tree: UnitTestTree;
+  const defaultOptions = {};
+  const lazyRoutePath = '/lazy-route.ts';
+  const lazyRoute = `
+    import { Route } from '@angular/router';
+    const routes: Array<Route> = [
+      {
+        path: '',
+        loadChildren: './lazy/lazy.module#LazyModule'
+      }
+    ];
+  `;
+
+  const lazyChildRoute = `
+    import { Route } from '@angular/router';
+    const routes: Array<Route> = [
+      {
+        path: '',
+        children: [{
+          path: 'child',
+          loadChildren: './lazy/lazy.module#LazyModule'
+        }]
+      }
+    ];
+  `;
+  const packageJson = {
```

Comment:
you can remove the below if you use 
```
    beforeEach(async () => {
      tree = new UnitTestTree(new EmptyTree());
      tree = await schematicRunner.runExternalSchematicAsync(
        require.resolve('../../collection.json'), 'ng-new',
        {
          name: 'migration-test',
          version: '1.2.3',
          directory: '.',
        },
        tree,
      ).toPromise();
    });
```

Instead of 
```
   beforeEach(() => {
      tree = new UnitTestTree(new EmptyTree());
      tree.create(packageJsonPath, JSON.stringify(packageJson, null, 2));
    });
```